### PR TITLE
Add operator and cert policy event test

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Ref: https://issues.redhat.com/browse/ACM-10146

### CODE REVIEW

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -443,6 +450,216 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
 		}, defaultTimeoutSeconds*2, 1).Should(Succeed())
 	})
+
+	It("Creates a policy with a compliant and non-compliant cert policy", func(ctx context.Context) {
+		const policyName = "cert-policy"
+		const certNs = "cert-policy-test-ns"
+		const certPath = "../resources/compliance_history/cert-policy.yaml"
+		const certSecret = "cert-secret"
+
+		now := time.Now().Add(-1 * time.Second).Format(time.RFC3339)
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:460]:
```suggestion
		now := time.Now().Format(time.RFC3339Nano)
```

File: test/integration/compliance_history_test.go [hunk 2]
```diff
@@ -443,6 +450,216 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
+				g.Expect(event1["event"].(map[string]interface{})["compliance"]).To(Equal("Compliant"))
+				g.Expect(event1["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("All evaluated certificates are compliant"))
+
+			}
+		}, 120, 1).Should(Succeed())
+	})
+
+	It("Creates a policy with a noncompliant and compliant operator policy", func(ctx context.Context) {
+		now := time.Now().Add(-1 * time.Second).Format(time.RFC3339)
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:564]:
```suggestion
		now := time.Now().Format(time.RFC3339Nano)
```

File: test/integration/compliance_history_test.go [hunk 3]
```diff
@@ -443,6 +450,216 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
+				g.Expect(err).ToNot(HaveOccurred())
+
+				event0 := events[0].(map[string]interface{})
+				g.Expect(event0["event"].(map[string]interface{})["compliance"]).To(Equal("NonCompliant"))
+				g.Expect(event0["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("CatalogSource 'invalid' was not found"))
+			}
+		}, 120, 1).Should(Succeed())
+
+		now = time.Now().Format(time.RFC3339)
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:624]: This helps with flakiness.
```suggestion
		now = time.Now().Format(time.RFC3339Nano)
```

File: test/integration/compliance_history_test.go [hunk 4]
```diff
@@ -443,6 +450,216 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
+			"op":"replace",
+			"path":"/spec/policy-templates/0/objectDefinition/spec/subscription/source",
+			"value":"redhat-operators"
+		}]`)
+		Expect(err).ToNot(HaveOccurred())
+		_, err = common.OcHub("patch", "policies.policy.open-cluster-management.io", policyName,
+			"-n", policyNS, "--type=json", "-p", `[{
+		"op":"replace",
+		"path":"/spec/remediationAction",
+		"value":"enforce"
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:638]: Is there clean up to remove the operator after it's installed?

Comment (yiraeChristineKim) [reply test/integration/compliance_history_test.go:638]: I remove operator-policy-ns namespace

**Review by mprahl — DISMISSED** [review]

**Comment by mprahl** [comment]
@yiraeChristineKim could you please rebase your PR?

**Review by mprahl — APPROVED** [review]

**Pull request merged**